eegnb/cli/__main__.py

```diff
@@ -73,14 +78,24 @@ def askforsigqualcheck():
             "Sorry, didn't recognize answer. "
             askforsigqualcheck()
     
+    def askforreportcheck():
+        do_sigqual = input("\n\nGenerate Report? (y/n). Recommend y \n")
+        if do_sigqual != 'y':
+            generatereport= False
```

Comment:
If y is recommended, then assume and make it default

```suggestion
        generatereport = input("\n\nGenerate Report? (Y/n): ").lower() != "n"
```